content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -52,6 +52,10 @@ spec:
 When deploying to a VM using [`istioctl workload entry configure`](/docs/setup/install/virtual-machine/), basic DNS proxying will be enabled by default.
 {{< /tip >}}
 
+{{< tip >}}
+When using the ambient data plane in Istio 1.24 or earlier, DNS Capture is not enabled by default. To enable DNS Capture for ambient set `values.cni.ambient.dnsCapture=true` during installation.
+{{< /tip >}}
+
```

Comment:
PTAL at https://deploy-preview-16230--preliminary-istio.netlify.app/latest/docs/ops/configuration/traffic-management/dns-proxy/

It says "This isn't enabled by default, here's how you enable it"

and now has a tip that says "this isn't enabled by default in 1.24 or earlier in ambient mode"

(a) is this now on by default for everyone in 1.25?  or only for ambient mode?
(b) can we reformat the instructions to say "Sidecar mode enablement", "Ambient mode enablement" etc
(c) can the sidecar mode enablement be as simple as the ambient mode enablement?